daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Ref.scala

```diff
@@ -13,7 +13,7 @@ object Ref {
 
   /* Encoding of byte array */
   type HexString = IdString.HexString
-  val HexString = IdString.HexString
+  val HexString: HexStringModule[Ref.IdString.HexString] = IdString.HexString
```

Comment:
```suggestion
  val HexString: IdString.HexString.type = IdString.HexString
```

Reply:
Thanks, you're officially smarter than IntelliJ. :smile: